packages/schematics/angular/library-entrypoint/index.ts

```diff
@@ -0,0 +1,93 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.dev/license
+ */
+
+import {
+  Rule,
+  SchematicsException,
+  Tree,
+  apply,
+  applyTemplates,
+  chain,
+  mergeWith,
+  move,
+  strings,
+  url,
+} from '@angular-devkit/schematics';
+import { JSONFile } from '../utility/json-file';
+import { relativePathToWorkspaceRoot } from '../utility/paths';
+import { buildDefaultPath, getWorkspace } from '../utility/workspace';
+import { ProjectType } from '../utility/workspace-models';
+import { Schema as LibraryOptions } from './schema';
+
+function updateTsConfig(packageName: string, ...paths: string[]) {
+  return (host: Tree) => {
+    if (!host.exists('tsconfig.json')) {
+      return host;
+    }
+
+    const file = new JSONFile(host, 'tsconfig.json');
+    const jsonPath = ['compilerOptions', 'paths', packageName];
+    const value = file.get(jsonPath);
+    file.modify(jsonPath, Array.isArray(value) ? [...paths, ...value] : paths);
+  };
+}
+
+export default function (options: LibraryOptions): Rule {
+  return async (host: Tree) => {
+    return async (host: Tree) => {
+      const workspace = await getWorkspace(host);
+      const project = workspace.projects.get(options.project);
+
+      if (!project) {
+        throw new SchematicsException(`Project "${options.project}" does not exist.`);
+      }
+
+      if (project?.extensions.projectType !== ProjectType.Library) {
+        throw new SchematicsException(
+          `Library entrypoint schematic requires a project type of "library".`,
+        );
+      }
+
+      const path = buildDefaultPath(project);
+      const libDir = `${path}/${options.name}`;
+      const pkgPath = `${project.root}/package.json`;
+
+      const pkg = host.readJson(pkgPath) as { name: string } | null;
+      if (pkg === null) {
+        throw new SchematicsException('Could not find package.json');
```

Comment:
Make the error message more actionable.

```suggestion
        throw new SchematicsException('Could not find "${pkgPath}".');
```